test/apiv2/01-basic.at

```diff
@@ -47,4 +47,19 @@ t GET info 200           \
   .DefaultRuntime=runc   \
   .MemTotal~[0-9]\\+
 
+# 2020-02-24 #5158 introduces a massive slowdown, 1-2s per request.
```

Comment:
I'd not leave this comment in the code.  I'd raise an issue though if not already done.

Reply:
#5311 already filed. I'll rephrase it; I just wanted something to explain the reasoning behind this test